src/index.ts

```diff
@@ -87,12 +87,16 @@ export const jestConfig: TSESLint.FlatConfig.Config = {
   },
 }
 
-export const javascriptConfig: TSESLint.FlatConfig.Config = {
```

Comment:
do we need this, isn't our typescript config covering the javascript files, those files could contain typing information as well.